docs/docs/contributing/code.mdx

```diff
@@ -61,6 +61,12 @@ cd libs/community
 
 ### Local Development Dependencies
 
+Install Poetry if you haven't already with:
+
+```bash
+pip install poetry
```

Comment:
removing this as it is slightly different from poetry's [installation docs](https://python-poetry.org/docs/#installing-with-pipx), which we link to above.